lib/stages/pulsarPostProcess.cpp

```diff
@@ -145,32 +154,32 @@ void pulsarPostProcess::main_thread() {
     uint64_t frame_fpga_seq_num = 0;     // sample starting the current input frame
     uint32_t current_input_location = 0; // goes from 0 to _samples_per_data_set
 
-    struct VDIFHeader vdif_header;
-    vdif_header.seconds = 0; // UD
-    vdif_header.legacy = 0;
-    vdif_header.invalid = 0;
-    vdif_header.data_frame = 0; // UD
-    vdif_header.ref_epoch = 36; // First half of 2018.
+    struct PSRHeader psr_header;
+    psr_header.seconds = 0; // UD
+    psr_header.legacy = 0;
+    psr_header.invalid = 0;
+    psr_header.data_frame = 0; // UD
+    psr_header.ref_epoch = 36; // First half of 2018.
     unix_offset = 1514764800;   // corresponds to 2018.01.01.0:0:0 in UTC
-    vdif_header.unused = 0;
+    psr_header.unused = 0;
     if (_timesamples_per_pulsar_packet == 3125) {
-        vdif_header.frame_len = 768;  //(6250-B data + 6-B pad + 32-B header)
-        vdif_header.log_num_chan = 1; // 2pol so ln2=1
+        psr_header.frame_len = 768;  //(6250-B data + 6-B pad + 32-B header)
+        psr_header.log_num_chan = 1; // 2pol so ln2=1
     } else if (_timesamples_per_pulsar_packet == 625) {
-        vdif_header.frame_len = 629;  // 5032-B
-        vdif_header.log_num_chan = 3; // ln8
+        psr_header.frame_len = 629;  // 5032-B
+        psr_header.log_num_chan = 3; // ln8
     }
-    vdif_header.vdif_version = 1;
+    psr_header.vdif_version = 1;
     char si[2] = {'C', 'X'};
-    vdif_header.station_id = (si[0] << 8) + si[1]; // Need to fomrally ask the Vdif community
-    vdif_header.thread_id = 0;                     // UD   freq
-    vdif_header.bits_depth = 3;                    // 4+4 bit so 4-1=3
-    vdif_header.data_type = 1;                     // Complex
-    vdif_header.edv = 0;
-    vdif_header.eud1 = 0; // UD: beam number [0 to 9]
-    vdif_header.eud2 = 0; //_psr_scaling from metadata
-    vdif_header.eud3 = 0; // Not used for now
-    vdif_header.eud4 = 0; // 16-b RA + 16-b Dec
+    psr_header.station_id = (si[0] << 8) + si[1]; // Need to fomrally ask the Vdif community
```

Comment:
```suggestion
    psr_header.station_id = (si[0] << 8) + si[1];
```
We aren't going to register this format.   We might register the new VDIF system, but that's going to be another part of the code.

Reply:
removed, and added a string describing the current use of `eud3`.